### PR TITLE
Roles to groups changed from json string to yaml

### PR DESCRIPTION
**Pull request by bojmil-armedia**

### CODE REVIEW

File: vagrant/provisioning/roles/arkcase-app/templates/server.yml [hunk 1]
```diff
@@ -43,7 +43,8 @@ wopi.plugin:
     domain: "{{ external_host }}"
     port: 443
 application:
-  rolesToGroups: "{{ roles_to_groups }}"
+  rolesToGroups:
+    {{ roles_to_groups | to_nice_yaml | indent(4) }}
```
Comment (david-oc-miller) [inline vagrant/provisioning/roles/arkcase-app/templates/server.yml]: I need to read up on all the Jinja2 filters... I don't remember seeing to_nice_yaml  or indent before.

File: vagrant/provisioning/arkcase-ce-facts.yml [hunk 2]
```diff
@@ -54,10 +54,25 @@ ldap_prefix: ""
...
 ldap_force_password_reset: yes
 
-# it takes some work to get Ansible to output the rolesToGroups value in the format 
-# needed by ArkCase.  Each role name and each group name have to be surrounded by \\\", 
-# and separated by a colon, and everything has to be one long line
-roles_to_groups: "{\\\"ROLE_ENTITY_ADMINISTRATOR\\\": \\\"{{ ldap_prefix }}ARKCASE_ENTITY_ADMINISTRATOR@ARKCASE.ORG\\\", \\\"ROLE_CONSUMER\\\": \\\"{{ ldap_prefix }}ARKCASE_CONSUMER@ARKCASE.ORG\\\", \\\"ROLE_SUPERVISOR\\\": \\\"{{ ldap_prefix }}ARKCASE_SUPERVISOR@ARKCASE.ORG,{{ ldap_prefix }}ARKCASE_ADMINISTRATOR@ARKCASE.ORG\\\", \\\"ROLE_ADMINISTRATOR\\\": \\\"{{ ldap_prefix }}ARKCASE_ADMINISTRATOR@ARKCASE.ORG,{{ ldap_prefix }}ARKCASE_ENTITY_ADMINISTRATOR@ARKCASE.ORG\\\", \\\"ROLE_EXTERNAL\\\": \\\"{{ ldap_prefix }}ARKCASE_EXTERNAL@ARKCASE.ORG\\\", \\\"ROLE_CONTRIBUTOR\\\": \\\"{{ ldap_prefix }}ARKCASE_CONTRIBUTOR@ARKCASE.ORG\\\"}"
+roles_to_groups:
+  ROLE_ENTITY_ADMINISTRATOR:
+  - "{{ ldap_prefix }}ARKCASE_ENTITY_ADMINISTRATOR@{{ ldap_user_domain | upper }}"
+  ROLE_CONSUMER:
```
Comment (david-oc-miller) [inline vagrant/provisioning/arkcase-ce-facts.yml:67]: somehow we need a guard on the version here, so if you are installing a version less than 3.3.3 it still uses the old ugly style.  Somewhere else in the plays I have some version checks.

File: vagrant/provisioning/roles/arkcase-app/templates/server.yml (hunk 1 above)
Comment (bojmil-armedia) [reply vagrant/provisioning/roles/arkcase-app/templates/server.yml]: I did spent 2-3 hours to come up with a way to format this one correctly.

File: vagrant/provisioning/arkcase-ce-facts.yml (hunk 2 above)
Comment (bojmil-armedia) [reply vagrant/provisioning/arkcase-ce-facts.yml:67]: OK, I'll fix it.

**Review by bojmil-armedia — COMMENTED** [review]
@david-oc-miller I made the changes. Please, check it out.

**Pull request merged**